mobile_verifier/src/rewarder.rs

```diff
@@ -457,15 +492,15 @@ async fn reward_poc(
         &boosted_hexes,
         &boosted_hex_eligibility,
         &poc_banned_radios,
-        reward_period,
+        &reward_info.epoch_period,
```

Comment:
I guess this is as good a place as any to ask the question.

Here we're passing only the `epoch_period`, and in a handful of other places we're passing the full `ResolvedSubDaoEpochRewardInfo` when we only use the `epoch_period` from it.

Do we want to be consistent and pass the full struct? Or continue to only pass `epoch_period` in places that don't use more information from `reward_info`?

Reply:
the approach I have taken is to pass the `ResolvedSubDaoEpochRewardInfo` struct to the top level or primary reward functions.  Thereafter call outs to sub functions use the existing function signature where possible.  There were a couple of places which didnt follow this patten and where the full struct was passed to sub functions unnecessarily, those are now fixed.

addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393